cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -830,6 +831,22 @@ public synchronized void executeProposals(Collection<ExecutionProposal> proposal
                             requestedIntraBrokerPartitionMovementConcurrency, requestedClusterLeadershipMovementConcurrency,
                             requestedBrokerLeadershipMovementConcurrency, requestedExecutionProgressCheckIntervalMs, replicaMovementStrategy,
                             isTriggeredByUserRequest, loadMonitor);
+      if (removedBrokers != null && !removedBrokers.isEmpty()) {
+        int count = 0;
+        int totalCount = 0;
+        for (ExecutionProposal proposal: proposals) {
+          Set<Integer> oldBrokers = proposal.oldReplicasBrokerIdSet();
+          Set<Integer> newBrokers = proposal.newReplicasBrokerIdSet();
+          if (!oldBrokers.equals(newBrokers)) {
+            // Only count the proposals that involve partition movement.
+            totalCount++;
+            if (oldBrokers.stream().anyMatch(removedBrokers::contains) || newBrokers.stream().anyMatch(removedBrokers::contains)) {
+              count++;
+            }
+          }
+        }
+        LOG.info("User task {}: {} of partition move proposals are related to removed brokers.", uuid, ((float) count) / totalCount);
```

Comment:
You may want to actually log count and total count here, to observe whether this only happens for large scale rebalance or not.